FWCore/Catalog/interface/FileLocator.h

```diff
@@ -53,11 +48,11 @@ namespace edm {
     /** Inverse rules are used to do the mapping from PFN to LFN*/
     ProtocolRules m_inverseRules;
 
-    std::string                 m_fileType;
-    std::string                 m_filename;
-    std::vector<std::string>    m_protocols;
-    std::string                 m_destination;
+    std::string m_fileType;
```

Comment:
many people like to align variable names.
The code is perhaps more readable that way.
So, this is a somewhat weak "-1" for this